pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1276,7 +1273,7 @@ def test_dt64arr_add_sub_relativedelta_offsets(self, box_with_array):
             ]
         )
         vec = tm.box_expected(vec, box_with_array)
-        vec_items = vec.squeeze() if box_with_array is pd.DataFrame else vec
```

Comment:
This is not about ArrayManager not supporting `squeeze`, this is just because I changed the dataframe from shape (1, 3) to (2, 3), so to get the first row as a Series, `squeeze` now no longer is suited in general (this is not related to ArrayManager vs BlockManager)